proto/staking/v1beta1/msg.proto

```diff
@@ -84,12 +74,11 @@ message MsgSignerUpdate {
             "github.com/maticnetwork/heimdall/types.ValidatorID",
         (gogoproto.customname) = "ID"
     ];
-    google.protobuf.Any new_signer_pub_key = 3
-        [(gogoproto.casttype) =
-             "github.com/maticnetwork/heimdall/types/common.PubKey"];
-    bytes tx_hash = 4
-        [(gogoproto.casttype) =
-             "github.com/maticnetwork/heimdall/types/common.HeimdallHash"];
+    google.protobuf.Any new_signer_pub_key = 3 [
```

Comment:
We thought of using string for pub_keys, isn't it? above (line 38) we just used singer_pub_key as a string.